src/Api.php

```diff
@@ -16,6 +17,8 @@
  */
 class Api {
 
+  protected $_base_url;
```

Comment:
I don't think starting the variable name with `_` follows coding standards. https://www.drupal.org/docs/develop/standards/coding-standards#naming

Reply:
I'll change that. D8 standards are joke though :)